rand_distr/README.md

```diff
@@ -19,7 +19,12 @@ It is worth mentioning the [statrs] crate which provides similar functionality
 along with various support functions, including PDF and CDF computation. In
 contrast, this `rand_distr` crate focusses on sampling from distributions.
 
-Unlike most Rand crates, `rand_distr` does not currently support `no_std`.
+If the `std` default feature is enabled, `rand_distr` uses the floating point
+functions from `std`. Otherwise, the floating point functions from `num_traits`
+and `libm` are used to support `no_std` environments.
```

Comment:
Isn't this not true right now? We don't seem to forward the `std` trait to num-traits, so would we even end up using the `std` implementations?

Reply:
ya we need to add `"rand-traits/std"` to this line: https://github.com/rust-random/rand/blob/3a03c9eb5350e03a3f540dba2ee34e0984f2c2c2/rand_distr/Cargo.toml#L24

Reply:
You mean "num-traits/std"?

This actually brings up another question though: *should* we prefer the `std` impls? This might affect portability of results.

Reply:
> You mean "num-traits/std"

Whoops, ya

> This actually brings up another question though: _should_ we prefer the `std` impls?

I feel like we should, if only because that's what num-traits does. Also, preffering `std` over libm is what I would expect. But I don't have that strong a feeling either way.

Reply:
> I feel like we should

Maybe; I don't have a strong opinion on it, though we should probably mention it [in the book](https://rust-random.github.io/book/portability.html) and maybe also try to gather data on where results may differ.

Reply:
Fair enough, I decided to forward our "std" feature as "num-traits/std", since this is the behavior I would expect.

Of course, this might technically be a value-breaking change. If we really want to have value stability for floating-point distributions, we might want to always use `libm`, because I am not sure that all implementations of the C standard math libraries produce the same results.